bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -78,6 +78,9 @@ <h2 class="section-title"><span>{{ _('Modern') }}</span></h2>
           </a>
         </p>
       </li>
+    {% set modern_iterator = 4 %}
+    {% if l10n_has_tag('firefox-features-send-tabs') %}
+      {% set modern_iterator = 5 %}
```

Comment:
You are totally right that this is a non-scalable fix. I do think re-examining how we do GA in this section is the next step here. I've asked Peter to give his thoughts in the bug.